src/components/molecules/Help/Help.tsx

```diff
@@ -1,72 +1,87 @@
-import React, { HTMLAttributes } from 'react';
+import React from 'react';
 import styled from 'styled-components';
-import Link from '../../atoms/Link/Link';
 import Text from '../../atoms/Text/Text';
 import Heading from '../../atoms/Heading/Heading';
 import FlexCol from '../../layout/FlexCol/FlexCol';
 import FlexRow from '../../layout/FlexRow/FlexRow';
-import { colors, grid } from '../../../constants';
+import { colors } from '../../../constants';
+import Card from '../../organisms/Card';
+import { buttonStyle } from '../../atoms/Button/Button';
+import Icon from '../../atoms/Icon/Icon';
+import { faEnvelope, faPhone } from '@fortawesome/free-solid-svg-icons';
 
 const HelpWrap = styled.div`
   background: ${colors.white};
-  padding: 80px 16px;
   box-sizing: border-box;
 `;
 
 const HelpContent = styled.div`
   margin: 0 auto;
-  max-width: 900px;
+  max-width: 700px;
 `;
 
-const HelpLinks = styled(FlexCol)`
-  margin-bottom: 24px;
-
-  @media (min-width: ${grid.breakpoints.l}px) {
-    margin-bottom: 0;
-  }
+const FlexColCenter = styled(FlexCol)`
+  text-align: center;
 `;
 
-const OpeningHoursWrapper = styled.div`
-  max-width: 350px;
+const ButtonAsLink = styled.a`
+  ${buttonStyle}
 `;
 
-export interface HelpProps extends HTMLAttributes<HTMLDivElement> {
-  email: string;
-}
-
-const Help: React.FC<HelpProps> = ({ email, ...rest }) => (
-  <HelpWrap {...rest}>
+const Help: React.FC = () => (
+  <HelpWrap className="pt-4">
     <HelpContent>
-      <FlexRow gutter={16}>
-        <FlexCol xs={12}>
-          <Heading as="h3" className="mb-6">
+      <FlexRow>
+        <FlexColCenter xs={12}>
+          <Heading as="h3" className="pb-6">
             We're here to help
           </Heading>
-        </FlexCol>
-      </FlexRow>
-      <FlexRow gutter={16}>
-        <HelpLinks xs={12} l={6}>
-          <Text className="mb-6">
-            <Link href={`mailto:${email}`}>{email}</Link>
-          </Text>
-          <Text className="mb-6">
-            <Link href="tel:020 7580 6060">020 7580 6060 </Link>
-            for loans
+          <Text className="pb-8" color={colors.grey}>
+            We can't take applications over the phone. <br />
+            UK residents only. Calls may be monitored or recorded
           </Text>
-          <Text>
-            <Link href="tel:020 7291 8331">020 7291 8331 </Link>
-            for investments
-          </Text>
-        </HelpLinks>
+        </FlexColCenter>
+      </FlexRow>
+      <FlexRow>
+        <FlexCol xs={12} l={6} className="pb-6">
+          <Card styling="primary">
+            <Card.Content>
+              <FlexRow>
+                <FlexColCenter xs={3} align="center">
+                  <Icon variant={faPhone} color={colors.brand} size="3x" />
+                </FlexColCenter>
+                <FlexCol xs={9}>
+                  <Card.Heading>020 7580 6060</Card.Heading>
+                  <Card.Text>Monday to Thursday (8am to 8pm), Friday (8am to 5pm)</Card.Text>
```

Comment:
Was there an instance during covid where our call center hours changed? If so i'd suggest we add a prop for timings which has these values as defaults that way if it happens again we wont be required to update the component library to make the change

Reply:
Mmm... Do you think it makes sense? I mean you won't have to upgrade the library but you still will have to manually add the time in every application

Reply:
Also the covid situation is something exceptional, right? I mean we haven't changed that in the past

Reply:
I guess we can release an old version as a patch fix in these scenarios